packages/forgae-utils/utils/forgae-utils.js

```diff
@@ -1,7 +1,14 @@
 require = require('esm')(module /*, options */) // use to handle es6 import/export 
+let axios = require('axios');
+const fs = require('fs');
+const path = require('path')
 const AeSDK = require('@aeternity/aepp-sdk');
 const Universal = AeSDK.Universal;
-const ContractCompilerAPI = AeSDK.ContractCompilerAPI;
+let rgx = /^include\s+\"([a-zA-Z+\d\w\/\.\-\_]+)\"/gmi;
+let dependencyPathrgx = /"([a-zA-Z+\d\w\/\.\-\_]+)\"/gmi;
+const mainContractsPathRgx = /.*\//g;
+let dependencies;
```

Comment:
should not be file/global variable, it should be declared inside function and pass to another or itself (if it is recursion)